### DISCUSSION TITLE
How to use Multiple Filters in Table Page?

### DISCUSSION THREAD

**Discussion by catrawalkar** [AntlerVC/firetable#424] [Support / Q&A] [answered]
I have a constant Filter. How can I apply more than 1 filter to the page??

I want that filter to be there every time. How can I change source code so that I can hardcode the filter in code?

**Comment by harinij** [comment, marked as answer]
This is now available in the recent releases, where you can apply a filter at table level to all users.

<img width="717" alt="Screen Shot 2022-07-05 at 6 58 24 pm" src="https://user-images.githubusercontent.com/307298/177290958-a5a83410-80ef-484a-955c-694fa8410f78.png">

**Comment by bofeiw** [comment]
Multiple filters is supported in the [Rowy 3.0 release](https://github.com/rowyio/rowy/releases/tag/v3.0.0).

![image](https://github.com/rowyio/rowy/assets/34177142/594adf17-4a81-4161-96ab-a7ccf458c2c4)
